configschema.json

```
@@ -307,6 +307,17 @@
         }
       },
       "required": ["enabled"]
+    },
+    "tiltify": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "enable": { "type": "boolean", "default": true },
```

```suggestion
        "enable": { "type": "boolean", "default": false },
```

I think it's better to make it false by default?